crates/core/c8y_api/src/utils.rs

```diff
@@ -13,6 +14,38 @@ pub mod bridge {
             Err(_err) => false,
         }
     }
+
+    pub fn is_c8y_bridge_established(message: &Message) -> bool {
+        match message.payload_str() {
```

Comment:
Why even check the payload, and validate it against all possible outcomes: 1 and 0? We could just check if the message is received on the health topic as we don't care about the value.